content/docs/tasks/security/vault-ca/index.md

```diff
@@ -105,6 +105,8 @@ certificate signing requests to Vault.
     on the testing Vault CA.
     To learn more about configuring a Vault CA for Kubernetes authentication and authorization,
     visit the [Vault Kubernetes `auth` method reference documentation](https://www.vaultproject.io/docs/auth/kubernetes.html).
+    The guide in [the link](https://evalle.xyz/posts/integration-kubernetes-with-vault-auth/) includes
+    detailed examples of configuring Vault to authenticate and authorize Kubernetes service accounts.
 
```

Comment:
The [Integration Kuberentes with Vault - auth](https://evalle.xyz/posts/integration-kubernetes-with-vault-auth/) post includes....

Reply:
Done.